tests/static-raw-html/.env

```
@@ -0,0 +1 @@
+tttt
```

```suggestion
TTTT=1234
```

`tttt` is definitely not a valid dotenv format 😂